packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -376,25 +376,33 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		);
 	}
 
-	private getBlobHandle(id: string): BlobHandle {
+	private getBlobHandle(localId: string, storageId: string | undefined): BlobHandle {
```

Comment:
It's a little easier to make `storageId` optional - this way we never need to pass in `undefined` when we call `getBlobHandle()`
```suggestion
	private getBlobHandle(localId: string, storageId?: string): BlobHandle {
```